Week3/homework/step2-2.js

```diff
@@ -1,23 +1,39 @@
-'use strict';
+('use strict');
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i); // push all nrs between the startIndex & the stopIndex
+  }
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 3 === 0) {
+      // apply a condition to check if the [i] is divided by 3
+      threeCallback(numbers[i]);
+    } else if (numbers[i] % 5 === 0) {
```

Comment:
One small thing, because you chain the if statement now the function outputs '10 can be divided by 5.', but not the '10 can be divided by 3.'. You can easily solve this by not using the `} else if` but putting the `if` on a new line, to make it two separate if statements.